operators/advanced_network/CMakeLists.txt

```diff
@@ -13,89 +13,60 @@
-  adv_network_kernels.cu
-  managers/adv_network_mgr.cpp
+# RX holoscan operator
+add_library(advanced_network_rx SHARED adv_network_rx.cpp)
+add_library(holoscan::ops::advanced_network_rx ALIAS advanced_network_rx)
+target_link_libraries(advanced_network_rx PUBLIC advanced_network_common holoscan::core)
+target_include_directories(advanced_network_rx
+    PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+        $<INSTALL_INTERFACE:include>
+)
+set_target_properties(advanced_network_rx PROPERTIES
+    OUTPUT_NAME "holoscan_op_advanced_network_rx"
+    EXPORT_NAME ops::advanced_network_rx
 )
 
-# Define targets for each manager type
-foreach(MGR IN LISTS ANO_MGR_LIST)
-    string(TOUPPER ${MGR} MGR_UPPER)
-    string(TOLOWER ${MGR} MGR_LOWER)
-    message(STATUS "Adding manager ${MGR_UPPER} to the configuration")
-    target_compile_definitions(advanced_network_common PUBLIC "ANO_MGR_${MGR_UPPER}=1")
-    target_link_libraries(advanced_network_common PRIVATE ano_manager_${MGR_LOWER})
-endforeach()
-
-target_include_directories(advanced_network_common PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/managers)
-
-add_library(advanced_network_rx SHARED adv_network_rx.cpp)
+# TX holoscan operator
 add_library(advanced_network_tx SHARED adv_network_tx.cpp)
-
-add_library(holoscan::advanced_network_rx ALIAS advanced_network_rx)
-add_library(holoscan::advanced_network_tx ALIAS advanced_network_tx)
-
-
-set_target_properties(advanced_network_common PROPERTIES CUDA_SEPARABLE_COMPILATION ON)
-set_target_properties(advanced_network_common PROPERTIES CUDA_RESOLVE_DEVICE_SYMBOLS ON)
-set_target_properties(advanced_network_common PROPERTIES CUDA_ARCHITECTURES "80;90")
-set_target_properties(advanced_network_rx PROPERTIES CUDA_ARCHITECTURES "80;90")
-set_target_properties(advanced_network_tx PROPERTIES CUDA_ARCHITECTURES "80;90")
-
-
-target_link_libraries(advanced_network_common PUBLIC holoscan::core)
-target_link_libraries(advanced_network_rx PUBLIC advanced_network_common)
-target_link_libraries(advanced_network_tx PUBLIC advanced_network_common)
-
-if(HOLOHUB_BUILD_PYTHON)
-    add_subdirectory(python)
-endif()
+add_library(holoscan::ops::advanced_network_tx ALIAS advanced_network_tx)
+target_link_libraries(advanced_network_tx PUBLIC advanced_network_common holoscan::core)
+target_include_directories(advanced_network_tx
+    PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+        $<INSTALL_INTERFACE:include>
+)
+set_target_properties(advanced_network_tx PROPERTIES
+    OUTPUT_NAME "holoscan_op_advanced_network_tx"
+    EXPORT_NAME ops::advanced_network_tx
+)
 
 # Installation
 install(
     TARGETS
-        advanced_network_common
         advanced_network_rx
         advanced_network_tx
+    EXPORT holoscan-networking-targets
+    COMPONENT advanced_network-cpp
+)
+install(
+    FILES
+        adv_network_rx.h
+        adv_network_tx.h
+    DESTINATION include/holoscan/operators/advanced_network
```

Comment:
Would be good to note somewhere the expected install include path for ANO/BNO headers if different from the HoloHub source tree. I.e. ANO apps have `#include "advanced_network/<header.hpp>"` but here it seems like the install interface will require `#include <holoscan/operators/advanced_network/<header.hpp>"` (if I'm reading correctly)

Reply:
I'm thinking I'll add `include/holoscan/operators/` as an install include dir do that the include still works and we still have it like other Holoscan operators, so both options would work.

Reply:
Had to even include `include/holoscan/operators/advanced_network`. not ideal but necessary to make it compatible with the #include in the holohub src code without changing them for the install tree.